bin/run_all_part_1.sh

```diff
@@ -20,7 +20,7 @@ root_dir=`cd "${current_dir}/.."; pwd`
 
 . ${root_dir}/bin/functions/color.sh
 
-for benchmark in `cat $root_dir/conf/benchmarks.lst`; do
```

Comment:
As this change is only for CI, can we keep the files under bin and conf like run_all.sh, benchmarks.lst unchanged, and add all new files in the CI folder?